modules/core/src/lib/layer-manager.js

```diff
@@ -310,12 +290,10 @@ export default class LayerManager {
         if (!oldLayer) {
           const err = this._initializeLayer(newLayer);
           error = error || err;
-          initLayerInSeer(newLayer); // Initializes layer in seer chrome extension (if connected)
```

Comment:
If we could leave a hook here, something like `LayerManager.callbacks.onInitialize`, it will allow users to add back similar functionalities using an external module.

Reply:
> If we could leave a hook here, something like LayerManager.callbacks.onInitialize, it will allow users to add back similar functionalities using an external module.

I like that proposal! I am playing with the idea of setting up some open source "dot-gl" libraries from Unfolded's side that could contain some of the more experimental code that is being cut from the core libraries, plus some new stuff that we are developing that would be complementary to the existing frameworks.

That way the vis.gl core libraries could go ahead focus on being really small and hardened, but also provide the extensibility hooks so that we can continue to build out more experimental and  advanced functionality (and if and when that matures we can always go back and make it official parts of the suite).